test_schemas/en/test_business.json

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
Probably could condense this down to just a single block

Reply:
It was taken from `test_introduction.json` as the original `mbs_0253.json` had multiple validation errors and I only want to test if schema loader is working.

Reply:
I'll remove this schema now and add `form_type` and `eq_id` to one of the test schemas.👍